### PR TITLE
Update `/tags/delSeries` to always propagate delete requests to other nodes

### PR DESCRIPTION
**Pull request by fitzoh**
This aligns the endpoint behavior with `/metrics/delete`, which ensures requests are forwarded to all nodes.

per @replay : 
>IIRC initially the propagate flag was necessary because when one cluster member received a delete call with propagate:true then it would fan it out to the same endpoint of the other cluster members, but in the sub-sequent calls it would set propagate:false to prevent a loop. now this intra cluster call is using a different endpoint /index/tags/delSeries

### CODE REVIEW

**Comment by Dieterbe** [comment]
LGTM. what does @replay say?

**Review by replay — CHANGES_REQUESTED** [review]
LGTM

I also tested it using the `docker-cluster-query` stack. I fed it 10 tagged metrics using fakemetrics, then stopped producing them, then deleted them one by one using this endpoint. It all worked fine, the request got fanned out correctly and there was no loop etc. 

I do think that we should add a note to breaking changes, because we know that there are users who use this endpoint regularly. Once this note is added then this is good to go.

**Comment by fitzoh** [comment]
Note added to changelog in 6dda57abad4decbca2c595e981d99ffcf9635692

**Review by replay — APPROVED** [review]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## breaking changes
 
+* as of `$VERSION` the `/tag/delSeries` no longer accepts a `propagate` parameter.
```
Comment (replay) [inline CHANGELOG.md]:
```suggestion
* as of `$VERSION` the `/tags/delSeries` no longer accepts a `propagate` parameter.
```

**Pull request merged**